dart/lib/src/default_integrations.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:async';
+
+import 'hub.dart';
+import 'protocol.dart';
+import 'sentry.dart';
+import 'sentry_options.dart';
+import 'throwable_mechanism.dart';
+
+/// integration that capture errors on the runZonedGuarded error handler
+Integration runZonedGuardedIntegration(
+  AppRunner callback,
```

Comment:
maybe we should adopt `appRunner` for the callback var name

Reply:
✅